tests/Riverline/DynamoDB/ConnectionTest.php

```diff
@@ -25,7 +25,7 @@ public function testConnection()
 	public function testConnectionFromDynamoDbClient()
 	{
 		$dynamoDbClient = $this->conn->getConnector();
-		$newConnection = Connection::CreateFromDynamoDbClient($dynamoDbClient);
+		$newConnection = new Connection($dynamoDbClient);
```

Comment:
NB: The `setUp` and `testConnection` functions for this will both fail. I'd probably argue this test class becomes redundant now.